### ISSUE TITLE
Position-dependent viscous friction model is unfinished

### ISSUE THREAD

**Issue by geezIIT** [geez0x1/CompliantJointToolbox#6]
Like the title states, the position-dependent viscous friction model is unfinished:
`model/nonlinear/viscous_pos_dep.m`

**Comment by geez0x1** [comment]
Deleted from master in 29b83595b35479225094e40afe27349e01d220f4, now has its own branch, to be developed in due time.

**Comment by joernmalzahn** [comment]
Hmm, this might go into the ripple model?

**Comment by geezIIT** [comment]
Yeah it may be possible to merge this with the more unified ripple model as a variant that depends on speed, I'll have to see.

**Comment by joernmalzahn** [comment]
Any update here?

**Comment by geez0x1** [comment]
I thought about how to include it in the `torque_ripple` function but it was a bit unnatural, since it doesn't just 'oscillate' around zero mean (w.r.t. speed) but actually has an offset, unlike the torque ripple sources. I don't think we have an immediate need for this model though.

**Comment by geez0x1** [comment]
The dev branch for this has been renamed to `dev_viscous_pos_dep`, to be consistent with other dev branches.
